server/zanata-security/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.zanata</groupId>
+    <artifactId>server</artifactId>
+    <version>4.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>zanata-security</artifactId>
+  <name>zanata-security</name>
+  <packaging>takari-jar</packaging>
+  <url>http://maven.apache.org</url>
```

Comment:
The url (zanata.org) should just be inherited from parent. I just noticed that the same thing is in gwt-editor.